lib/membrane_opus/decoder.ex

```diff
@@ -20,6 +19,11 @@ defmodule Membrane.Opus.Decoder do
                 at any supported sample rate. 48 kHz is recommended. For details,
                 see https://tools.ietf.org/html/rfc7845#section-5.1 point 5.
                 """
+              ],
+              channels: [
+                spec: 1 | 2,
+                default: 2,
+                description: "Expected number of channels"
```

Comment:
i was just backing out everything that used PacketUtils, but I see that you're using this to infer the channels instead of needing it as argument. I'll add that back.